TYPO3.Neos/Classes/TYPO3/Neos/EventLog/Domain/Repository/EventRepository.php

```diff
@@ -59,9 +59,7 @@ public function removeAll()
         $classMetaData = $this->entityManager->getClassMetadata($this->getEntityClassName());
         $connection = $this->entityManager->getConnection();
         $databasePlatform = $connection->getDatabasePlatform();
-        $connection->query('SET FOREIGN_KEY_CHECKS=0');
         $truncateTableQuery = $databasePlatform->getTruncateTableSql($classMetaData->getTableName());
         $connection->executeUpdate($truncateTableQuery);
-        $connection->query('SET FOREIGN_KEY_CHECKS=1');
```

Comment:
At least the tests all pass and manual trials showed that even if the events table has entries, truncating works fine. My guess is that there used to be references to other tables that actually had FK constraints, but that has been changed (and thus this hack is no longer needed).